example-tracing/README.md

```diff
@@ -0,0 +1,68 @@
+# Example Tracing
+
+## Overview
+
+This example illustrates how to enable tracing for a service deployed in Kyma. For demostration, it creates a [Go application](./order-front.go). This application uses [http-db-service](../http-db-service) for CRUD operations on orders.
+
+To understand how traces are propagated, see the [Go application](./order-front.go).  See the [example-tracing](./example-tracing.yaml) file to learn about port naming and setting the **app** label.
+
+## Prerequisites
+
+- A [Docker](https://docs.docker.com/install)
+- An environment in Kyma to which to deploy the example
+
+## Installation
+
+### Local installation
+
+1. Build the Docker image of the [Go application](./order-front.go). Run:
+    
+	```bash
+    docker build . -t order-front:latest
+    ```
+
+2. Run the image:
+    
+	```bash
+    docker run -p 8080:8080 order-front:latest
+    ```
+
+### Cluster installation
```

Comment:
I guess this paragraph should have steps numbered 1. - 3. instead of 5. - 3.? :)